samples/client/petstore/rust-reqwest/src/models/mod.rs

```diff
@@ -1,12 +1,12 @@
-mod api_response;
+pub mod api_response;
 pub use self::api_response::ApiResponse;
-mod category;
+pub mod category;
```

Comment:
I made a separate commit with an explanation: https://github.com/OpenAPITools/openapi-generator/pull/3895/commits/3c1d778a3840f82b648edebf0858f72487d4b30b

Essentially, if you want to be able to use generated enums that have the same property name, you need to be able to selectively import them to avoid duplicate type declaration.

Reply:
Ah. Good catch!